README.md

```diff
@@ -0,0 +1,244 @@
+# tox-pipenv2
+
+A reimplementation of [`tox-dev/tox-pipenv`](https://github.com/tox-dev/tox-pipenv) to address
+existing issues and rough edges encountered in a "strict-pinning" workflow.
+
+
+[![tox-pipenv on PyPI](https://img.shields.io/pypi/v/tox-pipenv2.svg)](https://pypi.python.org/pypi/tox-pipenv2)
+
+[![Test Package Compatibility](https://github.com/tox-dev/tox-pipenv/workflows/Test%20Package%20Compatibility/badge.svg)](https://github.com/tox-dev/tox-pipenv/actions)
+
+![PyPI downloads](https://img.shields.io/pypi/dw/tox-pipenv2)
+
+A tox plugin to replace the default use of pip with Pipenv.
+
+* Use `Pipfile` with multiple version of Python
+* Use `tox` to deploy strict-pinned dependencies from `Pipfile.lock.{envname}`
+  * Update lock files for multiple environments
+
+# Usage
+
+## `tox.ini`
+
+Add `tox-pipenv` to the top level `requires` list. Tox will automatically
+provision an environment with the given requirements.
+
+    [tox]
+    requires = tox-pipenv
+
+    [testenv]
+    command = {posargs}
+
+## `{toxinidir}/Pipfile`
+
+Use `pipenv` to create a `Pipfile` in the project directory, next to `tox.ini`.
+
+For tox environments without `deps`, this plugin will automatically use
+`pipenv install` to apply the `Pipfile` to the tox virtualenv.
+
+Environments that specify any `deps` will be _ignored_ by this plugin, as
+if `skip_pipenv = true` were set.
+
+### `{toxinidir}/Pipfile.lock.{envname}`
+
+If a lock file exists that matches the current tox environment name, this
+plugin will use `pipenv sync` to apply the lockfile into the tox virtualenv.
+
+# Regenerating the lock file
+
+When using lock files ("strict pinning"), the per-environment
+`Pipfile.lock.{envname}` must be regenerated whenever `Pipfile` changes or when
+updating dependency pins.
+
+```
+tox --recreate --pipenv-lock
+```
+
+Can be used to have this plugin explicitly re-lock the dependencies for each
+environment and copy the result to `{toxinidir}/Pipfile.lock.{envname}`.
+
+Subsequent tox invocations without `--pipenv-lock` will install the
+dependencies specified in the lockfile associated with the environment.
+
+To achieve repeatable deployments, it is recommended to keep the `Pipfile` and
+environment-specific lock files in version control.
+
+# Options
+
+## `skip_pipenv`
+
+_bool_. Specified per `[testenv]` section.
+
+If true, this plugin will not take any action in the environment.
+
+## `pipenv_install_cmd`
+
+_string_. Specified per `[testenv]` section.
+
+Override the sub-command passed to `pipenv` during the `install_deps` stage.
+
+By default, the plugin will use `sync` when a lockfile is present and `install`
```

Comment:
new diff will use `install --ignore-pipfile` instead of `sync`.

basically the idea is, if you have an environment-specific lock file, you definitely want to install from the lock file, _unless_

a) you specify `--pipenv-update` to update the lock file
b) you specify `TOX_PIPENV_INSTALL_OPTS="install --deploy"`, to check if the lock file is up-to-date

(there's a possibility that b) becomes `--pipenv-deploy`, but not sure if we need it)